src/apps/loan-list/list/ListHeader.tsx

```diff
@@ -11,7 +11,9 @@ const ListHeader: FC<ListHeaderProps> = ({ header, amount, children }) => {
     <div className="dpl-list-buttons">
       <h2 className="dpl-list-buttons__header">
         {header}
-        <div className="dpl-list-buttons__power">{amount}</div>
+        <span role="text" className="dpl-list-buttons__power">
```

Comment:
Apparently text is not a valid role?
From what I can read it should be a WAI-ARIA role as described in 5.4 definition of roles:
[5.4 Definition of Roles](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)